### PR TITLE
Styling 

### PR DESCRIPTION
**Pull request by ghost**
### PT
https://www.pivotaltracker.com/story/show/161012077

### Tasks
- Stying the landing page 
- Hold pictures in the grid

### CODE REVIEW

File: app/assets/stylesheets/application.sass [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    grid-gap: 30px
+    justify-content: space-around
+
+.grid-content
+    width: 350px
+    height: 350px
+    border: solid black
+    margin: auto
+
+    img
```
Comment (rupertlion) [inline app/assets/stylesheets/application.sass]: perhaps move this block above . grid-container (line 4) and then reduce indentation?

Comment (rupertlion) [reply app/assets/stylesheets/application.sass]: re: image block from line 16 onwards

Comment (Prozak8) [reply app/assets/stylesheets/application.sass]: Done

**Review by magnus-thor — APPROVED** [review]
Looks good

**Pull request merged**